Dev.md

```diff
@@ -0,0 +1,113 @@
+# Dev Setup
```

Comment:
Just a note for dev environment set up locally and in containers, I can remove this file if it's not needed.